lib/geth.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Manages the geth connection.
+ */
+
+const geth = require('geth')
+const muted = require('./muted').muted()
+
+const config = {
+  cache: 512,
+  bootnodes: 'enode://a24ac7c5484ef4ed0c5eb2d36620ba4e4aa13b8c84684e1b4aab0cebea2ae45cb4d375b77eab56516d34bfbd3c1a833fc51296ff084b770b94fb9028c4d25ccf@52.169.42.101:30303',
+  networkid: 4,
+  datadir: '$HOME/Library/Ethereum/rinkeby/',
+  rpc: null,
+  rpcapi: 'db,eth,net,web3,personal'
+}
+
+function connect (mineAddress) {
+  return new Promise((resolve, reject) => {
+    config.unlock = mineAddress
+    geth.start(config, {
+      stdout: (data) => {
+        // console.log('geth stdout:', data.toString())
+        if (data.toString().indexOf('Unlocking account') > -1) {
+          console.log('Enter password to unlock geth (input hidden):')
```

Comment:
Can we pass this as argument optionally? Forcing interactive startup kills docker 😌

Reply:
aw good point. we can pass as an argument to cli, or we can use --password to pull from a file. I can setup support for both.

Reply:
both are fine by me. We don't (shouldn't) support multiple options at this point. It's still kinda PoC-ish and offering multiple solutions to the same problem will just force us to break something later on. so let's not introduce it in the first place lol.

As long as it's automatable I'm happy. Both those solutions are also interchangable using a bit of bash magic. Pick whichever is easiest to implement.